service/src/main/kotlin/fi/espoo/evaka/assistanceneed/decision/AssistanceNeedDecision.kt

```diff
@@ -64,8 +66,12 @@ enum class AssistanceNeedDecisionLanguage {
 }
 
 data class AssistanceNeedDecisionEmployee(
+    @PropagateNull
     val employeeId: EmployeeId?,
-    val title: String?
+    val title: String?,
+    val name: String? = null,
+    val email: String? = null,
+    val phoneNumber: String?
```

Comment:
Defaulting to `null` would make sense for the `decisionMaker`, or maybe making a separate type for the decision maker as this person actually does not have a phone number.